changelog/3283.bugfix.rst

```diff
@@ -0,0 +1 @@
+Shifted the axis for calculation of ``edges_to_centers_nd`` about ``0`` for 1D array
```

Comment:
This changelog entry is misleading.  ```edges_to_centers_nd``` is not being changed.  How it's called elsewhere is being changed.  Suggested replacement:
```Stop crash when Animators' axes_ranges entry given as 1D array when data is >1D, i.e. as an independent axis.```